app/controllers/concerns/hyrax/collections_controller_behavior.rb

```diff
@@ -103,6 +103,8 @@ def create
       @collection.attributes = collection_params.except(:members)
       @collection.apply_depositor_metadata(current_user.user_key)
       add_members_to_collection unless batch.empty?
+      @collection.visibility = Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PRIVATE
```

Comment:
Should it be set to open if if @collection.dicoverable? is set to true here at create method?

As for the update method, I think that the default setting at creation should take care of it.